src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -9,56 +9,56 @@ import { logConsoleInfo } from "Utils/NotificationConsoleUtils";
 import { useSidePanel } from "hooks/useSidePanel";
 import React, { FunctionComponent, MouseEvent, useState } from "react";
 import { RightPaneForm, RightPaneFormProps } from "../RightPaneForm/RightPaneForm";
+import * as PriorityBasedExecutionUtils from "Utils/PriorityBasedExecutionUtils";
 
 export const SettingsPane: FunctionComponent = () => {
   const closeSidePanel = useSidePanel((state) => state.closeSidePanel);
   const [isExecuting, setIsExecuting] = useState<boolean>(false);
   const [pageOption, setPageOption] = useState<string>(
     LocalStorageUtility.getEntryNumber(StorageKey.ActualItemPerPage) === Constants.Queries.unlimitedItemsPerPage
       ? Constants.Queries.UnlimitedPageOption
-      : Constants.Queries.CustomPageOption,
+      : Constants.Queries.CustomPageOption
```

Comment:
We are now enforcing [trailing commas](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) in formatting. Please run `npm run format` correct these missing commas.

Reply:
It was some problem with the prettier module, the format issue is fixed now